meinberlin/assets/scss/components/_radio.scss

```diff
@@ -1,6 +1,10 @@
 
 .radio__input {
     @include sr-only;
+
+    &:disabled + .radio__text {
```

Comment:
@xi May I use the sibling-selector in meinBerlin? And what should happen to the colours?

Reply:
In general, I think that sibling selectors are ok, but should be avoided. In practice this means that you can use them, but are among the first things to be refactored.

In this specific case, I don't think this styling is useful. We do not generally change the styling of labels for disabled inputs. [Bootstrap](https://getbootstrap.com/docs/4.0/components/forms/#disabled-forms) also does not do this.

Reply:
Ok, so it's not clickable, but I can't see that before I click. That seems a bit annoying, but if that's the way to do it, I will of course do it like that.